e2e/_suites/ingest-manager/features/stand_alone_agent.feature

```diff
@@ -5,18 +5,30 @@ Feature: Stand-alone Agent
   with them
 
 @start-agent
-Scenario: Starting the agent starts backend processes
-  When a stand-alone agent is deployed
+Scenario Outline: Starting the <image> agent starts backend processes
+  When a "<image>" stand-alone agent is deployed
   Then the "filebeat" process is in the "started" state on the host
     And the "metricbeat" process is in the "started" state on the host
+Examples:
+| image   |
+| default |
```

Comment:
We use "default", although the code will only "massage" the service name when `image == "ubi"`